lib/modules/platform/bitbucket/index.ts

```diff
@@ -945,16 +946,66 @@ export async function createPr({
           },
         )
       ).body;
-      const pr = utils.prInfo(prRes);
+      pr = utils.prInfo(prRes);
       await BitbucketPrCache.addPr(
         bitbucketHttp,
         config.repository,
         renovateUserUuid,
         pr,
       );
-      return pr;
     }
   }
+
+  logger.debug(
+    {
+      repository: config.repository,
+      title,
+      base,
+      bbAutoResolvePrTasks: platformPrOptions?.bbAutoResolvePrTasks,
+    },
+    'Auto resolve PR tasks',
+  );
+  if (platformPrOptions?.bbAutoResolvePrTasks) {
```

Comment:
Is there any scenario where `pr` is null or undefined at this point?

Reply:
Looking at the code paths there shouldn't be a scenario where it's null, but to be defensive I've added a null check to the if statement

Reply:
I think it's a bit cleaner/simpler to keep `pr` as a  const, and move the `bbAutoResolvePrTasks` logic into a separate function, and call that in the appropriate places. I've added this as a separate commit. Let me know what you think.